pkg/driverbuilder/builder/builders.go

```diff
@@ -220,8 +220,8 @@ type GCCVersionRequestor interface {
 func defaultGCC(kr kernelrelease.KernelRelease) semver.Version {
 	switch kr.Major {
 	case 6:
-		if kr.Minor >= 10 {
-			return semver.Version{Major: 14, Minor: 2, Patch: 1}
+		if kr.Minor >= 9 {
```

Comment:
I used 9 because of https://bbs.archlinux.org/viewtopic.php?id=295952; it seems like archlinux 6.9+ needs gcc 14.